PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -326,6 +326,7 @@ def miniAOD_customizeMC(process):
     #GenJetFlavourInfos
     process.load("PhysicsTools.JetMCAlgos.HadronAndPartonSelector_cfi")
     task.add(process.selectedHadronsAndPartons)
```

Comment:
Apologies for not noticing this earlier but with the updates introduced in this PR, is anything consuming `selectedHadronsAndPartons` now? If not, this line can be removed.